### PR TITLE
Processing data models

### PR DESCRIPTION
**Pull request by c00p75**
# Blog App - processing data in models

## Objectives
- Created model classes for all entities as shown in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png).
- Set up associations between models.
- Use models to insert data
- Create custom methods for:
   - Users: Method returns the 3 most recent posts for a given user.
   - Posts: Method updates the posts counter for a user. Method returns the 5 most recent comments for a given post.
   - Comments: Method updates the comments counter for a post.
   - Likes: Method updates likes counter for a post.

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```
Comment (OyePriscilla) [inline app/models/comment.rb]: - [x] Please one more thing to do, kindly implement the method that updates the comments counter for a post as instructed in the project requirement. 🙏

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @c00p75 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Set up associations between models ✔️ 
Implemented methods for almost all the mode as expected ✔️ 

## Required Changes ♻️

- [ ] Kindly fix the Author section of your README file (linkedln to be precise), I got page not found. For your repo to be more engaging, its good that all your links redirects to appropraite pages. 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/comment.rb (hunk 1 above)
Comment (c00p75) [reply app/models/comment.rb]: I don't know how I missed this one. Thanks for the constructive feedback @OyePriscilla.

**Review by uwadonat — APPROVED** [review]
## Hi @c00p75,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good commit history ✔️
- Well organized and clean project ✔️
approvvv

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**